### ISSUE TITLE
Can add comment to DML like MySQL comment ？

### ISSUE THREAD

**Issue by yanghong** [ClickHouse/ClickHouse#26687]
Make sure to check documentation https://clickhouse.yandex/docs/en/ first. If the question is concise and probably has a short answer, asking it in Telegram chat https://telegram.me/clickhouse_en is probably the fastest way to find the answer. For more complicated questions, consider asking them on StackOverflow with "clickhouse" tag https://stackoverflow.com/questions/tagged/clickhouse 

If you still prefer GitHub issues, remove all this text and ask your question here.

**Comment by yanghong** [comment]
we always want to see some table comment and data comment from DML, but it seem CH doesn’t support.

**Comment by den-crane** [comment]
CH supports https://clickhouse.tech/docs/en/sql-reference/statements/create/table/#comment-table

```
CREATE TABLE xxx
(
    `name1` Int64
)
ENGINE = Log
COMMENT 'Comment'

Query id: 4148db3b-cedf-4ca8-849e-3a8342563e10

Connecting to localhost:9000 as user default.
Connected to ClickHouse server version 21.8.1 revision 54449.

Ok.

0 rows in set. Elapsed: 0.006 sec.
```
from version 21.6 https://github.com/ClickHouse/ClickHouse/blob/master/CHANGELOG.md
Implement table comments. Closes #23225. #23548 (flynn).
https://github.com/ClickHouse/ClickHouse/pull/23548
